### ISSUE TITLE
Update Project Profile: Civic Tech Jobs Add Kevin Yu

### ISSUE THREAD

**Issue by terrencejihoonjung** [hackforla/website#7556]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/civic-tech-jobs.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x] Find the `leadership` variable and add the following profile. 
```
  - name: Kevin Yu
    github-handle: kevin31yu
    role: Developer
    links:
      slack: https://hackforla.slack.com/team/U061GQQ56Q4
      github: https://github.com/kevin31yu
    picture: https://avatars.githubusercontent.com/kevin31yu
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Civic Tech Jobs page [^2]
- [x] Once your pull request is merged, go to the initiating ER [^3]
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/civic-tech-jobs
[^3]: initiating ER:  #7521

**Comment by HackforLABot** [comment]
Hello @NolaDodd, we appreciate you taking on issue #7556, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by HackforLABot** [comment]
Hi @srinipandiyan, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by srinipandiyan** [comment]
Availability for this week: Tuesday 7-8PM PST and Sunday 10-12PM PST

My estimated ETA for completing this issue: By Tuesday, before the dev meeting at 7PM PST.

**Comment by t-will-gillis** [comment]
Hi @srinipandiyan FYI- the note on the issue refers to moving the "initiating ER to Questions / In Review"- not moving this issue. I am moving this issue back to "QA". 

Please note: You no longer should move either this issue or the initiating ER- both are good where they are now.
